micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -501,6 +523,18 @@ public Timer timer(String name, String... tags) {
         return timer(name, Tags.of(tags));
     }
 
+    /**
+     * Measures the time taken for short tasks and the count of these tasks.
+     * @param name The base metric name
+     * @param tags Sequence of dimensions for breaking down the name.
+     * @return A new or existing timer.
+     * @since 1.16.0
+     */
+    public Timer timer(String name, Tags tags) {
```

Comment:
We could make this added API (and the equivalent counter/ltt API) non-public if we don't want to add API that's mostly sugar which could overwhelm users with options, but it seems harmless enough, and as mentioned in another comment, there are cases it can improve performance before JIT optimization.